app/javascript/lib/cell-faceting.js

```diff
@@ -112,6 +112,62 @@ function logFilterCells(t0Counts, t0, filterableCells, results, selection) {
   log('filter-cells', filterLogProps)
 }
 
+/**
+ * Determine if a cell satisfies any numeric filters
+ *
+ * @param {Number} d - A numeric datum; a numeric annotation value for a cell
+ * @param {Array<Array<String, *>>} numericFilters Filters for a numeric
+ *   facet. Each filter has an operator and a value.  Values can be a number
+ *   or an array of two numbers.
+ *
+ *   Example simple numeric filters:
+ *   - ["equals", 1.3]
+ *   - ["not equals", 1.3]
+ *   - ["greater than", 6]
+ *   - ["greater than or equal to", 6]
+ *   - ["less than", 6]
+ *   - ["less than or equal to", 6]
+ *   - ["between", [5, 42]] -- inclusive, i.e. 5 <= d <= 42
+ *   - ["not between", [5, 42]] -- inclusive, i.e. !(5 <= d <= 42)
+ *
+ *   Example compound numeric filters:
+ *   - [["between", [95, 100]], ["between", [1200, 1300]] -- (95 <= d <= 100) or (1200 <= d <= 1300)
+ *
+ *   Compound numeric filters could be used to e.g.:
+ *     - isolate the peaks of multimodal distributions (as in above concrete example)
+ *     - isolate the tails of distributions
+ *
+ *   TODO:
+ *   - Enable percentile filtering, i.e. beyond raw values.  Requires 1-time full sort, then trivial.
+ *
+ * @returns {Boolean} Whether cell datum passed any filters
+ */
+export function applyNumericFilters(d, numericFilters) {
+  for (let i = 0; i < numericFilters.length; i++) {
+    const [operator, value] = numericFilters[i]
+    if (operator === 'equals') {
```

Comment:
Non-blocking: would a `switch` statement be cleaner here?  Functionality is the same, but I always like `case [foo]` over a string of `else if`...

Reply:
I tried this, and while it's slightly cleaner, in our session yesterday we saw it's also slower, so opted to keep `if`s.